pkg/services/authz/zanzana/schema/folder.fga

```diff
@@ -37,35 +37,40 @@ type folder
     define parent: [folder]
     define org: [org]
 
-    define create: [user, team#member, role#assignee] or create from parent or folder_create from org
-    define read: [user, team#member, role#assignee] or read from parent or folder_read from org
-    define write: [user, team#member, role#assignee] or write from parent or folder_write from org
-    define delete: [user, team#member, role#assignee] or delete from parent or folder_delete from org
-    define permissions_read: [user, team#member, role#assignee] or permissions_read from parent or folder_permissions_read from org
-    define permissions_write: [user, team#member, role#assignee] or permissions_write from parent or folder_permissions_write from org
+    # Action sets
+    define view: [user, team#member, role#assignee]
+    define edit: [user, team#member, role#assignee]
+    define admin: [user, team#member, role#assignee]
 
-    define dashboard_create: [user, team#member, role#assignee] or dashboard_create from parent or dashboard_create from org
-    define dashboard_read: [user, team#member, role#assignee] or dashboard_read from parent or dashboard_read from org
-    define dashboard_write: [user, team#member, role#assignee] or dashboard_write from parent or dashboard_write from org
-    define dashboard_delete: [user, team#member, role#assignee] or dashboard_delete from parent or dashboard_delete from org
-    define dashboard_permissions_read: [user, team#member, role#assignee] or dashboard_permissions_read from parent or dashboard_permissions_read from org
-    define dashboard_permissions_write: [user, team#member, role#assignee] or dashboard_permissions_write from parent or dashboard_permissions_write from org
+    define create: [user, team#member, role#assignee] or edit or admin or create from parent or folder_create from org
+    define read: [user, team#member, role#assignee] or view or edit or admin or read from parent or folder_read from org
```

Comment:
I'm not sure I'm reading this correctly, but does this mean `read = [ view or edit or admin on this folder] or [read parent folder] or [folder_read in org]`? Basically just making sure that `from parent` doesn't apply to the action sets as well.

Reply:
yes, correct, `from parent` applies to one relation, so if you want to apply it to every relation, you should do it explicitly, like `view from parent or edit from parent...`